pkg/segment/writer/metrics/metricssegment.go

```diff
@@ -690,27 +691,61 @@ func ExtractInfluxPayload(rawCSV []byte, tags *TagsHolder) ([]byte, float64, uin
 				}
 			}
 
-			for _, value := range field_set {
-				kvPair := strings.Split(value, "=")
-				key := kvPair[0]
-				value = kvPair[1]
-				if key == influx_value {
-					fltVal, err := strconv.ParseFloat(value, 64)
-					if err != nil {
-						return nil, 0, 0, fmt.Errorf("failed to convert value to float! %+v", err)
-					}
-					dpVal = fltVal
+			for _, metricValueSet := range field_set {
+				kvPair := strings.Split(metricValueSet, "=")
+				metricName := kvPair[0]
```

Comment:
add a check here `if len(kvPair) < 2` then continue, else this is going to crash if someone sends bad data

Reply:
do the error counting as well

Reply:
We can get the errors count from doing `len(errors)`? I am actually storing the list/slice of errors.